src/components/BusinessForm/BusinessForm.jsx

```diff
@@ -87,59 +62,42 @@ const BusinessForm = ({ pending, pendingData, setBackButtonClicked }) => {
   const [city, setCity] = useState('');
   const [state, setState] = useState('');
   const [zip, setZip] = useState('');
-  const [country, setCountry] = useState('');
   const [website, setWebsite] = useState('');
+  const [countrycode, setCountryCode] = useState('');
   const [phone, setPhone] = useState('');
   const [email, setEmail] = useState('');
   const [howHeard, setHowHeard] = useState('');
   const [registrationSuccess, setRegistrationSuccess] = useState(false);
   const navigate = useNavigate();
-  const toast = useToast();
-  const [businessHours, setBusinessHours] = useState(
-    daysOfWeek.reduce(
-      (acc, day) => ({
-        ...acc,
-        [day]: {
-          start: { hour: '9', minute: '00', ampm: 'AM' },
-          end: { hour: '5', minute: '00', ampm: 'PM' },
-        },
-      }),
-      {},
-    ),
-  );
-
+  const [businessHours, setBusinessHours] = useState('');
+  const [checkedAddedInfo, setCheckedAddedInfo] = useState({});
+  const [webNotes, setWebNotes] = useState('');
+  const [published, setPublished] = useState(false);
+  const [webDateInit, setWebDateInit] = useState('');
+  const [internalNotes, setInternalNotes] = useState('');
+  const [fphPhone, setFPHPhone] = useState('');
+  const [vendorType, setVendorType] = useState('');
+  const [serviceRequest, setServiceRequest] = useState(false);
+  const { backend } = useBackend();
   useEffect(() => {
-    if (pending) {
+    if (edit) {
       fillOutPendingData();
-      navigate(`/AdminDashboard/${status}/${pendingData.id}`);
     }
-  });
-
-  const handleTimeChange = (day, period, value) => {
-    setBusinessHours({ ...businessHours, [day]: { ...businessHours[day], [period]: value } });
-  };
-
-  const handleZipChange = e => {
-    setZip(e.target.value);
-  };
-
-  const { backend } = useBackend();
-  const handleSubmit = async e => {
-    e.preventDefault();
+  }, [backend]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'edit' and 'fillOutPendingData'. Either include them or remove the dependency array.